pkg/agent/datamodel/types_test.go

```diff
@@ -2057,12 +2056,19 @@ func TestKubernetesConfigGetOrderedKubeletConfigString(t *testing.T) {
 					"--pod-manifest-path":            "/etc/kubernetes/manifests",
 					"--node-status-report-frequency": "5m0s",
 					"--cgroups-per-qos":              "true",
+					"--image-gc-high-threshold":      "80",
+					"--image-gc-low-threshold":       "60",
 					"--kubeconfig":                   "/var/lib/kubelet/kubeconfig",
 					"--keep-terminated-pod-volumes":  "false",
 				},
 			},
-			expected:              alphabetizedString,
 			expectedForPowershell: alphabetizedStringForPowershell,
+			CustomKubeletConfig: &CustomKubeletConfig{
```

Comment:
For #1, Line 2044 covers it.
For #2, the order should not impact anything since I just appended the new property in the struct in line 2034. I have updated the order